### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by phasnox** [aesmail/kaffy#149]
**Versions Used**
Kaffy: 0.9.0
Phoenix: 1.5.4
Elixir: 1.10.4

**What's actually happening?**
Error when executing list_action on all items. 

**What should happen instead?**
No error page should be displayed, and items updated

**Screenshots**
![kaffy_error](https://user-images.githubusercontent.com/3498059/91930861-87c8d180-eca7-11ea-8d71-5615d0308c7c.gif)

## Error trace
```
deps/kaffy/lib/kaffy/resource_query.ex:59: value `["", "2", "1"]` in `where` cannot be cast to type {:in, :id} in query:
from p0 in DecatIo.Profiles.Profile,
  where: p0.id in ^["", "2", "1"],
  select: p0

```

## Solution
The error is on [this line](https://github.com/aesmail/kaffy/blob/master/lib/kaffy_web/controllers/resource_controller.ex#L348)

The problem is that when all items are selected, an empty id is sent e.g. `["", "2", ""]`. The quick solution is to change that line to:

```
    ids = Map.get(params, "ids", "") |> String.split(",") |> Enum.filter(&(&1 != ""))
```

to filter the empty string.

**Comment by aesmail** [comment]
Hey @phasnox,
Thanks for the report.
This is actually fixed in #129 and will ship in the next version.
Let me know if `master` doesn't fix the issue.

**Comment by phasnox** [comment]
Thanks @aesmail ! Yes confirmed it fixed in master!
